### PR TITLE
Translation update

### PR DESCRIPTION
**Pull request by lbk2907**

### CODE REVIEW

**Comment by bear101** [comment]
@lbk2907 Please don't run Qt's lupdate on a branch. It will be impossible to merge if someone else does the same.

**Comment by lbk2907** [comment]
Are I just need to directly push the translation update if have any to the master branch? And you can merge this for now for this. I’ll create another pull request if I found any error or words that I need to correct it. Again, sorry about that. I’m new in Git and GitHub. I still learning about it.

**Comment by lbk2907** [comment]
Wait a sec. Did you said don’t run something? I think I didn’t run anything. I just open the .ts file using Notepad++ to edit the translation file and push it with new branch called “TranslationUpdate” instead of directly push it to the master branch yesterday. Anyway, I’m sorry if I have miss something. Do let me know so that I know where my mistake and I’ll more careful next time.

File: Client/qtTeamTalk/languages/Chinese_Traditional.ts [hunk 1]
```diff
@@ -3060,11 +3061,6 @@ Should these settings be applied?</source>
         <translation type="obsolete">檔案%1包含%2設定資訊
 要套用這些設定嗎?</translation>
     </message>
-    <message>
-        <location filename="../mainwindow.cpp" line="6008"/>
-        <source>Video device hasn&apos;t been configured properly. Check settings in &apos;Preferences&apos;</source>
-        <translation>視訊裝置未適當地設定. 請檢查&apos;偏好設定&apos;中的設定</translation>
```
Comment (bear101) [inline Client/qtTeamTalk/languages/Chinese_Traditional.ts:3066]: Hasn't the translation been deleted here?
<translation>視訊裝置未適當地設定. 請檢查&apos;偏好設定&apos;中的設定</translation>
now it's:
<translation type="unfinished"></translation>

File: Client/qtTeamTalk/languages/Croatian.ts [hunk 2]
```diff
@@ -3628,11 +3629,6 @@ Možeš je preuzeti na sljedećoj stranici:
         <source>Specify new nickname</source>
         <translation>Odredi novi nadimak</translation>
     </message>
-    <message>
-        <location filename="../mainwindow.cpp" line="6008"/>
-        <source>Video device hasn&apos;t been configured properly. Check settings in &apos;Preferences&apos;</source>
```
Comment (bear101) [inline Client/qtTeamTalk/languages/Croatian.ts:3633]: Also here the translation has been deleted.

File: Client/qtTeamTalk/languages/Danish.ts [hunk 3]
```diff
@@ -2966,8 +2966,9 @@ You can download it on the page below:<byte value="xd"/>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="4276"/>
+        <location filename="../mainwindow.cpp" line="6008"/>
         <source>Video device hasn&apos;t been configured properly. Check settings in &apos;Preferences&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Videoenhed kunne ikke konfigureres korrekt. Undersøg indstillingerne i &apos;Indstillinger&apos;</translation>
```
Comment (bear101) [inline Client/qtTeamTalk/languages/Danish.ts:2971]: Here the translation, however, looks like it was updated correctly.

**Comment by lbk2907** [comment]
That was the duplicate after I fix the string typo a few days ago.

**Comment by bear101** [comment]
But in Chinese_Traditional.ts there's no longer a translation for the item:
<source>Video device hasn&apos;t been configured properly. Check settings in &apos;Preferences&apos;</source>

Line 3066 should be moved to 2474 in Chinese_Traditional.ts. Shouldn't it?

**Comment by lbk2907** [comment]
Did I just miss it. How can I add back the missing line? Or you able to do that after you merge this?

**Comment by bear101** [comment]
I cannot fix it after a merge. Then the translation will be gone. I would suggest you try and revert your changes and then redo the corrections.

**Comment by lbk2907** [comment]
Ok. I going to push the correction. Sorry.

**Comment by lbk2907** [comment]
Done. I already add back that 2 missing line.

**Comment by bear101** [comment]
Thanks! I'll merge it now.

**Comment by lbk2907** [comment]
You’re welcome.

**Pull request merged**